README.md

```diff
@@ -631,6 +631,26 @@ const MyControl = SAPClass.extend('MyControl', {
 
 **CAUTION** The plugin does not currently search for 'metadata' or 'renderer' properties inside the constructor. So don't apply Babel's class property transform plugin before this one if you have metadata/renderer as instance properties (static properties are safe).
 
+### Comments
+
+In case of defining *copyright* comments in your source code (detected by a leading `!`), the plugin ensures to include it also as leading comment in the generated file, e.g.:
```

Comment:
Maybe mention that these "*copyright* comments" also need to be at the very beginning of the file.

Reply:
Well, this I can accept... 😁